### PR TITLE
Publicize: Remove '@since' and '@module publicize' tags from Publicize package

### PR DESCRIPTION
**Pull request by danielpost**
When porting the Publicize module over to its own package, some `@since` and `@module publicize` tags were ported over as well. These don't make sense in the context of the Publicize package, so we're removing them.

#### Changes proposed in this Pull Request:

* Remove all old `@since` tags from PHP DocBlocks in the Publicize package
* Similarly, remove all `@module publicize` tags.

#### Jetpack product discussion

p1655287019368319-slack-C02JJ910CNL

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* N/A

### CODE REVIEW

**Review by jeherve — COMMENTED** [review]
In the past, we opted to leave those in, but update them, like so: #20766

Could we do the same here?

**Comment by danielpost** [comment]
@jeherve That seems sensible, will do!

**Comment by spsiddarthan** [comment]
@jeherve, This change https://github.com/Automattic/jetpack/commit/9bdd5dca7ea8af3b64fe9bb140ddfc6cefe7eeac was merged to the master/trunk branch on the 19th of April. I wanted to add a `@since` tag for the publicize package based on this change.  

I am struggling a little bit to find out the correct way to determine the first publicize package version which contains this change. Any thoughts on how I can find out the package version?

**Comment by jeherve** [comment]
> I am struggling a little bit to find out the correct way to determine the first publicize package version which contains this change. Any thoughts on how I can find out the package version?

Check this post for scripts that should help you there:
p9dueE-3cJ-p2

File: projects/packages/publicize/src/class-keyring-helper.php [hunk 1]
```diff
@@ -110,8 +110,6 @@ private static function api_url( $service = false, $params = array() ) {
 		/**
 		 * Filters the API URL used to interact with WordPress.com.
 		 *
-		 * @since 2.0.0
```
Comment (spsiddarthan) [inline projects/packages/publicize/src/class-keyring-helper.php:113]:
```
@since 0.1.0
@since-jetpack 2.0.0
```

**Comment by spsiddarthan** [comment]
Closing this in favour of https://github.com/Automattic/jetpack/pull/24856, where  @gmjuhasz has implemented the changes, will request a review there.

**Pull request closed without merging**